packages/database-mongo/package.json

```diff
@@ -32,13 +32,13 @@
     "@accounts/types": "^0.30.0",
     "@accounts/mongo-password": "^0.30.0",
     "@accounts/mongo-sessions": "^0.30.0",
+    "@types/mongodb": "3.5.33",
```

Comment:
We don't add the types in the dependencies to not impact non-typescript projects.
What is the error with yarn pnp?

Reply:
`IndexOptions`, which is exported by `mongodb`, is used in the signature of one of the public interfaces:

https://github.com/accounts-js/accounts/blob/3c2264db9fa121677201c85730625ee4f27ba54c/packages/database-mongo/src/mongo.ts#L63

Therefore `IndexOptions` is used in the generated type definitions.

Yarn PnP does not allow hoisting (as it is ambiguous) and so an error is thrown when the type definitions try to import `mongodb` (in actual fact, it is only `@types/mongodb` that are needed). This can be resolved by enabling `skipLibCheck`, however this should not be a requirement to use this package.

> We don't add the types in the dependencies to not impact non-typescript projects.

I think in most cases this is ok, as they are not needed at compile time for other packages. However there are a small number of cases (such as this), where they are. Additionally, many packages ship with their own type definitions anyway, so I'm not sure there is much to gain for non-typescript users by not including `@types/mongodb`.

Reply:
Ok can we just unpin the version then? `"@types/mongodb": "^3.5.33",`

Reply:
Done